sites/platform/static/files/fetch/examples/relationships/mongodb

```diff
@@ -4,8 +4,11 @@
     "service": "mongodb36",
     "ip": "169.254.150.147",
     "hostname": "blbczy5frqpkt2sfkj2w3zk72q.mongodb36.service._.eu-3.platformsh.site",
-    "cluster": "rjify4yjcwxaa-master-7rqtwti",
     "host": "mongodb.internal",
+    "cluster": "rjify4yjcwxaa-master-7rqtwti",
```

Comment:
please do not expose potential IPs ;) 

for such config, please use: 
- IP: 123.456.78.90
-  "hostname": "azertyuiopqsdfghjklm.mongodb36.service._.eu-1.platformsh.site",